Gopkg.toml

```diff
@@ -53,9 +53,51 @@ required = [
 # See here:
 #  - https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/116#discussion_r262599059
 #  - https://github.com/kubernetes-sigs/controller-runtime/pull/349
+
+[[constraint]]
+  name="sigs.k8s.io/controller-runtime"
+  version="v0.1.10"
+
 [[constraint]]
+  name="sigs.k8s.io/controller-tools"
+  version="v0.1.1"
+
+# For dependency below: Refer to issue https://github.com/golang/dep/issues/1799
+[[override]]
```

Comment:
I would highly advise against `override` deps. It will cause several problems. I had discussed this with @justaugustus as well.

Reply:
I have this PR so that we can import the latest k8s the right way : https://github.com/kubernetes-sigs/controller-tools/pull/164

Reply:
Thanks Tariq, removed all overrides